src/main/java/io/github/moulberry/notenoughupdates/options/NEUConfig.java

```diff
@@ -129,6 +132,10 @@ public void executeRunnable(int runnableId) {
 			case 14:
 				editOverlay(activeConfigCategory, OverlayManager.fishingSkillOverlay, skillOverlays.fishingPosition);
 				return;
+			case 15:
+				String command = NotEnoughUpdates.INSTANCE.config.misc.fariySoul ? "/neusouls on" : "/neusouls off";
+				ClientCommandHandler.instance.executeCommand(Minecraft.getMinecraft().thePlayer, command);
```

Comment:
Confused: Is there a reason to use a command for this, but not for `setTrackFairySouls`? this seems fine, still.

Reply:
I did that for two reasons:
1. It provides consistent behavior between the pre-existing command and using the menu system to toggle the option.
2. The command can fail and I didn't see an obvious, lightweight way to show an error in the menu system and there was no reason to duplicate the error handling code.